cmd/registry/controller/parser.go

```diff
@@ -130,7 +131,7 @@ func extendDependencyPattern(
 
 func resourceNameFromParent(
 	resourcePattern string,
-	parent string) (ResourceName, error) {
+	parent string) (resourceName, error) {
```

Comment:
Nit: This seems like it can be reduced to one line now, since it's shorter than `ValidateManifest` below.

`func resourceNameFromParent(resourcePattern, parent string) (resourceName, error)`

Reply:
Done.